qt/pubnub_qt.cpp

```diff
@@ -530,6 +530,23 @@ pubnub_res pubnub_qt::publish(QString const& channel, QString const& message)
 }
 
 
+pubnub_res pubnub_qt::publish(QString const& channel, QString const& message, publish_options& opt)
+{
+    QMutexLocker lk(&d_mutex);
+    d_method = pubnubSendViaGET;
```

Comment:
With provided way to set publish options, this feel redundant:
```suggestion
```

Reply:
You're right. I just copy-pasted the whole function on the beginning and forgot to adjust that. Thanks!